### PR TITLE
Introduce VoyageAI module

### PR DESCRIPTION
**Pull request by fzowl**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: modules/text2vec-voyageai/clients/voyageai.go [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	return &ent.VectorizationResult{
+		Text:       input,
+		Dimensions: len(resBody.Data[0].Embeddings),
+		Vectors:    vectors,
+	}, nil
+}
+
+func (v *vectorizer) getVoyageAIUrl(ctx context.Context, baseURL string) string {
+	passedBaseURL := baseURL
+	if headerBaseURL := v.getValueFromContext(ctx, "X-VoyageAI-Baseurl"); headerBaseURL != "" {
```
Comment (antas-marcin) [inline modules/text2vec-voyageai/clients/voyageai.go:156]: you need to change it to `X-Voyageai-Baseurl` to make it work

File: modules/text2vec-voyageai/clients/voyageai.go [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+	}
+	// try getting header from GRPC if not successful
+	if apiKey := modulecomponents.GetValueFromGRPC(ctx, key); len(apiKey) > 0 && len(apiKey[0]) > 0 {
+		return apiKey[0]
+	}
+	return ""
+}
+
+func (v *vectorizer) getApiKey(ctx context.Context) (string, error) {
+	if apiKey := v.getValueFromContext(ctx, "X-VoyageAI-Api-Key"); apiKey != "" {
```
Comment (antas-marcin) [inline modules/text2vec-voyageai/clients/voyageai.go:183]: you need to change it to `X-Voyageai-Api-Key` - bc that is how golang's http library capitalizes headers

File: modules/text2vec-voyageai/clients/voyageai.go [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+func (v *vectorizer) getApiKey(ctx context.Context) (string, error) {
+	if apiKey := v.getValueFromContext(ctx, "X-Voyageai-Api-Key"); apiKey != "" {
+		return apiKey, nil
+	}
+	if v.apiKey != "" {
+		return v.apiKey, nil
+	}
+	return "", errors.New("no api key found " +
+		"neither in request header: X-Voyageai-Api-Key " +
```
Comment (antas-marcin) [inline modules/text2vec-voyageai/clients/voyageai.go:190]: here you can leave a nice form `X-VoyageAI-Api-Key`

File: test/acceptance_with_go_client/named_vectors_tests/named_vectors_test.go [hunk 4]
```diff
@@ -118,7 +118,7 @@ func composeModules() (composeModules *docker.Compose) {
 		WithText2VecTransformers().
 		WithText2VecOpenAI().
 		WithText2VecCohere().
-		WithGenerativeOpenAI().
+	    WithGenerativeOpenAI().
```
Comment (antas-marcin) [inline test/acceptance_with_go_client/named_vectors_tests/named_vectors_test.go:121]: please fix the indentation

**Comment by fzowl** [comment]
I agree with CLA

**Pull request merged**